topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/faqs/host_filtering_filter.md

```diff
@@ -0,0 +1,10 @@
+---
+title: What does `^.*Gallus|Homo|Bos.*$` mean?
+box_type: question
+layout: faq
+contributors: [EngyNasr]
+---
+
+It is an expression that matches everything that has words Gallus OR Homo OR Bos (c.f. Regular expression)
```

Comment:
Add a link to that, what is regular expression and how to find an example to write a regular expression

Reply:
For this you could refer to the general Regex FAQ that we have: https://training.galaxyproject.org/training-material/faqs/galaxy/analysis_regular_expressions.html

(and add to it if needed)